packages/dx-react-grid-bootstrap3/src/templates/table-band-header/banded-header-cell.test.jsx

```diff
@@ -3,21 +3,25 @@ import { shallow } from 'enzyme';
 import { BandedHeaderCell } from './banded-header-cell';
 
 describe('BandedHeaderCell', () => {
+  const defaultProps = {
+    component: () => <div />,
+    className: 'custom-class',
```

Comment:
Why is className a default prop?

Reply:
Because this class is used for each test.